chain/pool/src/lib.rs

```diff
@@ -105,8 +115,14 @@ impl TransactionPool {
 
     /// Reintroduce transactions back during the chain reorg
     pub fn reintroduce_transactions(&mut self, transactions: Vec<SignedTransaction>) {
+        let mut dropped_transactions = 0;
         for tx in transactions {
-            self.insert_transaction(tx);
+            if self.insert_transaction(tx).is_err() {
+                dropped_transactions += 1;
+            }
+        }
```

Comment:
Or if you want to be a hax0r, the following should work:
```rust
let dropped = transactions.into_iter().map(|tx| self.insert_transaction(tx)).filter(Result::is_err).count();
```

Reply:
I can, but I don't want to :)